src/aavegotchi/gotchi/gotchi-lending-ended-handler.ts

```diff
@@ -56,7 +56,7 @@ export function handleGotchiLendingEnded(event: GotchiLendingEnded): void {
 
   // update rental
   currentRental.expiration_date = event.block.timestamp;
```

Comment:
Since the get NFT Roles endpoint is going to break anyways. We can take this opportunity to refactor all the fields in Rental and Rental Offer to camel case.
```suggestion
  currentRental.expirationDate = event.block.timestamp;
```